### DISCUSSION TITLE
Some JPG images will resize, some made with Photoshop won't

### DISCUSSION THREAD

**Discussion by PetersOtto** [datenstrom/community#822] [Report a bug] [answered]
Hello everybody!

I have some problems with the image extension. Some images (.jpg) will resize, some won't. I made some test. I use Photoshop. When I use "export", yellow dont resize the image. When I use "save as", yellow resize the image. 

As well I found an image in the internet that won't resize from yellow. When I open it with Photoshop and use "save as", yellow will resize it. 

Is that a bug or is there an explanation for that behavior?

Best regards

Update: I use image extension `VERSION = "0.8.18"`

**Comment by markseuffert** [comment]
Could be a problem with image detection. Can you provide some example images to reproduce it?

**Comment by PetersOtto** [comment]
Yes, of course. The images with "export" in the filename do not work. I prepared this images with Photoshop export function.
[pizza-test-images.zip](https://github.com/datenstrom/yellow/files/10069359/pizza-test-images.zip)

On the following screenshot you will see the result on my LAMP-Server:
![yellow-images-resize-test](https://user-images.githubusercontent.com/52236754/203372881-5e192dc5-f18c-4bbb-891d-aaa10dec812a.png)

Let me know if you need more images.

**Comment by markseuffert** [comment, marked as answer]
Thank you for the bug report. I updated the [core extension](https://github.com/annaesvensson/yellow-core). There was a bug in the image detection. The JPG detection was written for images with the two most common JPG markers `APP0/JFIF` and `APP1/Exif`, but it ignored images with `APP14/Adobe`.

Should be fixed now, please try it out.

**Comment by PetersOtto** [comment]
It works like a charm :) With my own images and with the images I found in the internet.

Thank you for help and for developing/maintaining YellowCMS.
